tests/examples/test_examples.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#pylint: disable=no-member, too-many-locals, too-many-branches, no-self-use, broad-except, lost-exception, too-many-nested-blocks, too-few-public-methods, invalid-name
+"""
+    This file tests and ensures that the examples run correctly.
```

Comment:
it does not ensure the correctness of the example. it only performs a sanity check of the example. can you please change this.

Reply:
The discussion on the dev-list suggest that we start small just to check if examples are running fine. The individual examples test will have to take care of what correctness atrribute they want to check and should be taken care by the test owners. This PR just provide a template for putting tests for examples. The test owners are free to have their own helper functions and tests which can be generalised later.  As we see the tests because some examples do inference, some do training and everything has a different form of output , it cannot be generalised at this point. 

Please feel free to add additional helper functions over here if you feel it can be used across multiple examples.

Reply:
I will change the wordings here that the examples run without compilation errors if that makes more sense